nectar/src/bitcoin/bitcoind.rs

```diff
@@ -494,7 +496,7 @@ mod test {
 
         let network = client.network().await.unwrap();
 
-        assert_eq!(network, Network::Regtest)
+        assert_eq!(network, ledger::Bitcoin::Regtest)
```

Comment:
Did you consider bringing `ledger::Bitcoin` into scope with an alias
```
use ledger::Bitcoin as Network
```
Its a bit of a shame to have ta say `ledger::Bitcoin` when we are in a file that is solely bitcoin, (we have context from the filename already).